docs/architecture/project-configuration.md

```diff
@@ -108,11 +108,12 @@ Here's what happens in the Upstream actor
 **Legend**
 
 * ProjStateCh - Project State Channel
+* UpstreamProjSrc - Upstream Project Source
```

Comment:
Can we name these a little more human-readable, like `UpstreamSource` and `StateChannel`?

Reply:
ok, fixed